Script/AtomicEditor/editor/Editor.ts

```diff
@@ -41,14 +45,10 @@ class Editor extends Atomic.ScriptObject {
         this.subscribeToEvent("ExitRequested", (data) => this.handleExitRequested(data));
 
         this.subscribeToEvent("ProjectLoaded", (data) => {
-
-            Atomic.editorMode.preferences.registerRecentProject(data.projectPath);
-
+            prefs.registerRecentProject(data.projectPath);
```

Comment:
looks much better, I am wondering if we should have prefs use a static constructor to instantiate the singleton instead of new'ing it here is an example of a static constructor, which could new the instance:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/AtomicEditor/ui/inspector/InspectorUtils.ts#L4